README.md

```diff
@@ -14,15 +14,21 @@ Turn complex GraphQL queries into optimized database queries.
 pip install graphql-compiler
 ```
 
+## Quick Overview 
+
+The GraphQL compiler turns read-only queries written in GraphQL syntax to different 
+query languages. Through the GraphQL compiler, users can write powerful queries that uncover 
+deep relationships in the data while not having to worry about the underlying database query 
+language.
+
+Furthermore, the GraphQL compiler validates queries through the use of a GraphQLSchema object 
```

Comment:
`GraphQLSchema object` uses the internal name of the type i.e. an implementation detail. Let's refer to it everywhere as `GraphQL schema` instead.